test/test_ssl.py

```diff
@@ -372,3 +373,65 @@ def test_ssl_replication(pg, bouncer, cert_dir):
     # physical rep
     connect_args["replication"] = "true"
     bouncer.psql("IDENTIFY_SYSTEM", **connect_args)
+
+
+def test_servers_no_disconnect_on_reload_with_no_tls_change(bouncer):
+    bouncer.default_db = "pTxnPool"
+
+    with bouncer.cur() as cur:
+        # change nothing and RELOAD
+        bouncer.admin("RELOAD")
+
+        with bouncer.log_contains(
+            r"pTxnPool.*closing because: database configuration changed", 0
```

Comment:
Maybe we could test that the list of backend connections itself (maybe measured from PG itself?) remains constant? The log message test to show it's 0 makes me a bit nervous about false green tests.

Reply:
That's a good idea. We can assert on both. The log assertions seemed to be conventional in this suite so I think there's some confidence in the approach but I like also asserting on connections from PG.

Reply:
Added [assertions](
f14b97d) to address this feedback